docs/content/framework/systems/Transfers/MultiAppCopyTransfer.md

```diff
@@ -1,9 +1,17 @@
-<!-- MOOSE Object Documentation Stub: Remove this when content is added. -->
-#MultiAppCopyTransfer
+# MultiAppCopyTransfer
+
 !devel /Transfers/MultiAppCopyTransfer float=right width=auto margin=20px padding=20px background-color=#F8F8F8
 
 !description /Transfers/MultiAppCopyTransfer
 
+## Description
+The MultiAppCopyTransfer allows for copying variables (both [nonlinear](auto::/Variables/Overview) and [auxiliary](auto::/AuxVariables/Overview)) between [MultiApps](auto::/MultiApps/Overview). All types of
+variables, including higher order, elemental, and nodal are supported. The only limitiation is that the
+meshes in the master and sub application must be identical.
```

Comment:
You get an error if the meshes are different. I am using number of nodes and elements to define equivalent, I am not sure if there is something better. @jwpeterson is there a way to compare two meshes, I don't recall seeing operator()== in MeshBase.